script/gencode

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-files="Sources/Container.Arguments Sources/SynchronizedResolver.Arguments Sources/Resolver Sources/ServiceEntry.TypeForwarding"
```

Comment:
This wasn't functioning and hasn't been run in a while, I only committed the changes for the files I modified.